playbooks/generic/facts.yml

```diff
@@ -9,4 +9,6 @@
     - role: osism.commons.facts
 
 - name: Import gather-facts play
-  ansible.builtin.import_playbook: gather-facts.yml
+  hosts: "{{ hosts_facts|default(hosts_default_group|default('all')) }}"
```

Comment:
Looks like this is not possible at all: syntax-check[specific]: this task 'ansible.builtin.import_playbook' has extra params, which is only allowed in the following modules. Looks like the hosts are already configurable in https://github.com/osism/ansible-playbooks/blob/main/playbooks/generic/gather-facts.yml (it's

Reply:
Okay, my bad :-) I close this PR.